skglm/penalties/separable.py

```diff
@@ -380,6 +380,10 @@ def value(self, w):
         """Compute the value of L0_5 at w."""
         return self.alpha * np.sum(np.abs(w) ** 0.5)
 
+    def derivative(self, w):
+        """Compute the element-wise derivative."""
+        return 1. / (2. * np.sqrt(np.abs(w)) + 1e-12)
```

Comment:
yes. I usually use `np.finfo(float).eps` which is more consistent, but it is not JIT-compilable. Maybe a simpler solution would consist in passing `eps=1e-12` as derivative. WDYT?

Reply:
I don't think it solves the duplication problem.
I would proceed as follows:
```python
# small number added to a denominator to prevent it from being zero
EPS_DERIVATIVE = 1e-12
```

Reply:
I feel like it's a bit overkill to add a const variable in a large module like this. The PR should be finalized now. No strong opinions. Feel free to add it (cc @mathurinm )

Reply:
Yes I agree, it feels overkill for now. One less variable to expose